packages/openapi-spec-builder/src/openapi-spec-builder.ts

```diff
@@ -30,6 +29,13 @@ export function anOperationSpec() {
   return new OperationSpecBuilder();
 }
 
+/**
+ * Custom extensions can use arbitrary type as the value,
+ * e.g. a string, an object or an array.
+ */
+// tslint:disable-next-line:no-any
+export type ExtensionValue = any;
+
 export interface Extendable {
   [extension: string]: ExtensionValue;
 }
```

Comment:
`Extendable` should be `Extensible`.

Reply:
We can use `ISpecificationExtension` instead of `Extendable`. However, openapi3-ts does not come with any alias type similar to our `ExtensionValue`, that's why I thought it's better to preserve both our types.

OTOH, this is not a big deal. I think the benefits of using `ISpecificationExtension` are worth it.

I have updated the code accordingly, see a9e4ab914e.